kairon/shared/analytics/analytics_pipeline_processor.py

```
@@ -144,4 +143,27 @@ def add_event_log(event_id, bot, user, status, exception, pipeline_name, callbac
+                    logger.error(f"EmailActionConfig not found for bot={bot}, action_name={action_name}")
+                    continue
+                try:
+                    CallbackScriptUtility.send_email(
+                        email_action.action_name,
+                        from_email=email_action.from_email.value,
+                        to_email=email_action.to_email.value[0],
+                        subject=email_action.subject,
+                        body=email_action.response,
+                        bot=email_action.bot
+                    )
+                except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)